coderedcms/models/page_models.py

```diff
@@ -489,7 +489,7 @@ def get_index_children(self):
                 classifier=self.index_order_by_classifier,
                 # Reverse ManyToMany of `coderedpage.classifier_terms`.
                 coderedpage=models.OuterRef("pk"),
-            )
+            )[:1]
```

Comment:
The subquery is by default in the drag and drop order, so the first Term should be the highest "priority" term.

`.first()` doesn't work here because the OuterRef requires us to not evaluate the QS outside of a subquery. I tried that solution first.